client/components/boards/boardHeader.jade

```diff
@@ -117,6 +117,9 @@ template(name="boardChangeTitlePopup")
     label
       | {{_ 'title'}}
       input.js-board-name(type="text" value=title autofocus)
+    label
+      | {{_ 'description'}}
+      textarea.js-board-desc= description
```

Comment:
Hey that’s another feature! I won’t ask you to put it in a separate commit because I understand it would be painful, but could you just mention it in the commit in case (just in case we have some issue with it latter and we want to explore the history).